zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLQueryParserTest.scala

```diff
@@ -396,36 +396,125 @@ class AQLQueryParserTest extends Specification {
     }
 
     "translate a count cardinality query without group by" in {
-      val group = GroupStatement(Seq.empty, Seq(aggrCount))
-      val query = new Query(dataset = TwitterDataSet, groups = Some(group))
+      val globalAggr = GlobalAggregateStatement(aggrCount)
+      val query = new Query(dataset = TwitterDataSet, globalAggr = Some(globalAggr))
       val result = parser.parse(query, schema)
-      val expected =
-        """
-          |count (for $t in dataset twitter.ds_tweet return $t)
-        """.stripMargin
-      ok
+      removeEmptyLine(result) must_== unifyNewLine(
+        """count
```

Comment:
I think I made another mistake about the return value. Since every aggregation statement requires an `as`, we need to ship a record instead a value. The correct AQL string should be
```\count
  
            
  Write  Preview
I think I made another mistake about the return value. Since every aggregation statement requires an `as`, we need to ship a record instead a value. The correct AQL string should be
```
```

I think I made another mistake about the return value. Since every aggregation statement requires an `as`, we need to ship a record instead a value. The correct AQL string should be
```
{ "count": count( for $d in dataset aaa return $d.id) }
```